src/main/java/dk/cngroup/lentils/repository/StatusRepository.java

```diff
@@ -16,6 +16,7 @@
     Status findByTeamAndCypher(Team team, Cypher cypher);
     List<Status> findByTeam(Team team);
     Boolean existsStatusByCypherAndTeam(Cypher cypher, Team team);
+    Boolean existsStatusByCypherAndTeamAndCypherStatusEquals(Cypher cypher, Team team, CypherStatus cypherStatus);
```

Comment:
Změnil bych to, ať se vrací primitivní typ boolean s malým b, jestli by to šlo :)

Reply:
jasně, změním - a nevíš, jestli na metoda předtím (ř. 18) má z nějakého důvodu Boolean? Přiznám se, že jsem to dělala podle ní a moc jsem u toho neuvažovala...

Reply:
A jo, toto jsem přehlédl, i když je to o řádek výš.
Žádný podstatný důvod to asi nemá. Ty boxed datové typy se používají spíš, když s tím potřebuješ pracovat jako s objektem, předávat reference atd. Jinak to moc smysl nemá.